.eslintrc.js

```diff
@@ -60,9 +60,10 @@ module.exports = {
   },
   overrides: [
     {
-      files: ['*test*', 'setupTests.js'],
+      files: ['src/tests/**/*'],
       rules: {
         'react/prop-types': 'off',
+        'import/no-import-module-exports': 'off',
```

Comment:
this was failing linting for the added `globalSetup.js` file, which I am using to set the timezone. This ensures snapshots are rendered with the same time zone every time and don't fail tests, since this patch adds date and time to the search results

Reply:
In general we decided against one-off disabling like this as it can get confusing if used in too many places. I try to avoid just turning a rule off and instead configure it if possible, but again this is for test files only.

The rule here is disabled for test files only. I did look into `cross-env` from your other comment, we could add that later on and wouldn't need the global setup file anymore.

Reply:
Ah yeah, I see now that it's only for tests, then it's less a concern for me. I'm not used to these `overrides` properties much as it's fairly new in eslint :-)